analyzer/src/analysis/value/interprocedural/sequential/function_fixpoint.cpp

```diff
@@ -76,7 +77,19 @@ FunctionFixpoint::FunctionFixpoint(
       _checkers(checkers),
       _exit_invariant(make_bottom_abstract_value(ctx)),
       _return_stmt(nullptr),
-      _logger(logger) {}
+      _logger(logger),
+      _namer()
+{
+  if (_ctx.opts.trace_ar_statements) {
+    this->_namer =
+        std::unique_ptr<ar::Namer>(new ar::Namer(entry_point->body()));
```

Comment:
Use `std::make_unique`, here and below
```suggestion
    this->_namer = std::make_unique<ar::Namer>(entry_point->body());
```